frontend/src/metabase/visualizations/visualizations/IFrameViz/IFrameViz.tsx

```diff
@@ -61,17 +62,21 @@ export function IFrameViz({
     return (
       <IFrameEditWrapper>
         <Stack h="100%" spacing="sm">
-          <Group align="center">
-            <Text fw="bold">{t`Paste your snippet here`}</Text>{" "}
-            <Button
-              disabled={!iframeOrUrl}
-              compact
-              ml="auto"
-              variant="filled"
-              style={{ pointerEvents: "all" }}
-              onClick={onTogglePreviewing}
-              onMouseDown={e => e.stopPropagation()}
-            >{t`Done`}</Button>
+          <Group align="center" noWrap>
+            <Text fw="bold" style={{ overflow: "hidden" }}>
+              <Ellipsified>{t`Paste your snippet here`}</Ellipsified>
```

Comment:
Could we use `truncate` or `lineClamp` on the Text element to avoid having to use `Ellipsified` here? Not a huge issue, but I think with Mantine some of this is built into that `Text` component.